vision/cloud-client/detect/vision_async_batch_annotate_images.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample
+#   ("LongRunningPromise",  "vision_async_batch_annotate_images")
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-vision
+
+# sample-metadata
+#   title: Async Batch Image Annotation
+#   description: Perform async batch image annotation
+#   usage: python3 samples/v1/vision_async_batch_annotate_images.py \
+#     [--input_image_uri \
+#     "gs://cloud-samples-data/vision/label/wakeupcat.jpg"] \
+#     [--output_uri "gs://your-bucket/prefix/"]
```

Comment:
Should we drop all this now that it technically isn't auto-generated?

Reply:
Done.